src/streamlink_cli/main.py

```diff
@@ -144,17 +170,21 @@ def create_title(plugin=None):
         if _game is None:
             _game = DEFAULT_STREAM_METADATA["game"]
         
+        #python 2 unicode handling
```

Comment:
What was the problem here? If `args.title` is unicode it should be fine right?

Reply:
python's string.format doesn't want to format unless I put that there.

Reply:
I didn't, but I think I would need the "maybe_decode" function you mentioned if I want it to be prettier than what's written here.

Reply:
Yeah, like that :)